.github/workflows/test-private.yaml

```diff
@@ -84,7 +87,7 @@ jobs:
       - name: Install Keycloak
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
-          helm install keycloak bitnami/keycloak -n keycloak --create-namespace --version 21.2.0
+          helm install keycloak bitnami/keycloak -n keycloak --create-namespace --version 21.2.0 --set image.repository=bitnamilegacy/keycloak --set postgresql.image.repository=bitnamilegacy/postgresql
```

Comment:
This change already happened in main branch, likely need to rebase to ensure the diff is correct?